docs/api/api/api.txt

```diff
@@ -1944,6 +1943,21 @@ POST /staging_excluded_requests/<request>/<project>
 DELETE /staging_excluded_requests/<request>
   Stop excluding this request
 
+=== Staged Requests
+
+<staging_project>: The name of the staging project, e.g. home:user1:Staging:A
+
+GET /staging_projects/<staging_project>/staged_requests
+  Get all staged requests of a staging project.
+XmlResult: staged_requests
+
+POST /staging_projects/<staging_project>/staged_requests
```

Comment:
what is missing here is the format expected. It's something with <number>1</number> from what I saw

Reply:
after the last changes, the routes look more like:

```
/staging/:staging_main_project_name/staging_projects/:staging_project_name/staged_requests(.:format)
```